doc/examples/reconst_mapmri.py

```diff
@@ -161,21 +163,23 @@
 fig = plt.figure(figsize=(10, 5))
 ax1 = fig.add_subplot(1, 3, 1, title=r'RTOP - Laplacian')
 ax1.set_axis_off()
-ind = ax1.imshow(mapfit_laplacian_aniso.rtop()[:, 0, :].T,
-                 interpolation='nearest', origin='lower', cmap=plt.cm.gray,
-                 vmin=0, vmax=5e7)
+rtop_laplacian = np.array(mapfit_laplacian_aniso.rtop()[:, 0, :].T,
+                          dtype=float)
+ind = ax1.imshow(rtop_laplacian, interpolation='nearest',
+                 origin='lower', cmap=plt.cm.gray, vmin=0, vmax=5e7)
 
 ax2 = fig.add_subplot(1, 3, 2, title=r'RTOP - Positivity')
 ax2.set_axis_off()
-ind = ax2.imshow(mapfit_positivity_aniso.rtop()[:, 0, :].T,
-                 interpolation='nearest', origin='lower', cmap=plt.cm.gray,
-                 vmin=0, vmax=5e7)
+rtop_positivity = np.array(mapfit_positivity_aniso.rtop()[:, 0, :].T,
+                           dtype=float)
+ind = ax2.imshow(rtop_positivity, interpolation='nearest',
+                 origin='lower', cmap=plt.cm.gray, vmin=0, vmax=5e7)
 
 ax3 = fig.add_subplot(1, 3, 3, title=r'RTOP - Both')
 ax3.set_axis_off()
-ind = ax3.imshow(mapfit_both_aniso.rtop()[:, 0, :].T,
-                 interpolation='nearest', origin='lower', cmap=plt.cm.gray,
-                 vmin=0, vmax=5e7)
+rtop_both = np.array(mapfit_both_aniso.rtop()[:, 0, :].T, dtype=float)
+ind = ax3.imshow(rtop_both, interpolation='nearest', origin='lower',
+                 cmap=plt.cm.gray, vmin=0, vmax=5e7)
```

Comment:
`vmin` and `vmax` are incorrect. This wrong range is the reason for your black image. Can you just remove this 2 parameters and let  `matplotlib` manage it